src/task/script.js

```diff
@@ -1,4 +1,25 @@
 const API_BASE_URL = window.API_BASE_URL;
+const suggestedUsers = [];
+const allUser = [];
+const category = document.getElementById('category');
+
+category.addEventListener('change', async () => {
+  try {
+    showSubmitLoader();
```

Comment:
are we disabling the select tag so that the user cannot change it while there is a request on the way.
or add a debounce function so that it will wait for lets say 0.5s before it makes the call and you can disable the select tag after the 0.5s

Reply:
as per the discussion, I am already disabling that